### ISSUE TITLE
Vue ui run serve not view dashboard

### ISSUE THREAD

**Issue by vallemar** [vuejs/vue-cli#5339]
### Version
4.2.3

### Environment info
```
  System:
    OS: Windows 10 10.0.18363
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor
  Binaries:
    Node: 13.12.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.4 - C:\Users\valle\AppData\Roaming\npm\npm.CMD
  Browsers:
    Edge: 44.18362.449.0
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
1- Launch: vue ui
2- Go to the tasks tab
3- Serve
4- Execute task

### What is expected?
The dashboard with all the information should come out.

### What is actually happening?
Only the tab appears: Output.

---
The build tab if it appears well, dashboard and analyzer.
In any other vue project if the serve tab works for me.
I don't have any configuration in the project to modify vue ui, so I don't know what is happening.

I have tried to configure vue-cli-service with the minimum configuration (vue-cli-service serve) and add multiple parameters but it does not work
![image](https://user-images.githubusercontent.com/15719383/78025029-9fe3a000-7359-11ea-9108-167f65d16835.png)

**Comment by acidiney** [comment]
any error log in console?

**Comment by vallemar** [comment]
No, since I open vue ui until I get to the (run serve) screen, the console does not give any error.

It only happens to me with this project, all the others work well

**Comment by gtsiolis** [comment]
@vallemar are you still seeing this?
